examples/display_model.py

```diff
@@ -25,9 +25,10 @@ def main():
     parser = ParlaiParser(True, True)
     parser.add_argument('-n', '--num-examples', default=10)
     opt = parser.parse_args()
+    opt['datatype'] = 'valid'
 
     # Create model and assign it to the specified task
-    agent = create_agent(opt)
+    agent = create_agent(opt, requireModelExists=True)
```

Comment:
some models don't use a model file, e.g. ir_baseline -- will it still work?

Reply:
Oh that's a great point, we can drop this part of the patch.